pkg/config/config_test.go

```diff
@@ -131,7 +132,8 @@ func Test_ExtractConfigData(t *testing.T) {
  seedNode: true
  segreaderNode: true
  metareaderNode: true
- segFlushIntervalSecs: 1200
+ idleWipFlushIntervalSecs: 1200
```

Comment:
I guess this will get rejected and will default to the value that we allow

Reply:
Yes